www/docs/en/dev/guide/platforms/windows/index.md

```diff
@@ -23,55 +23,30 @@ toc_title: Windows
-within Visual Studio. Regardless of [which](../../overview/index.html#development-paths) approach you take, you need to
-install the Visual Studio SDK, as described below.
-
-Developers wishing to target Windows Phone 8 should use the wp8 platform,
-see [Windows Phone 8 Platform Guide](../wp8/index.html) for details (Warning, the wp8 platform is deprecated).
-
-Cordova WebViews running on Windows rely on Internet Explorer 11 (Windows 8.1 and Windows Phone 8.1) as
-their rendering engine, so as a practical matter you can use IE's
-powerful debugger to test any web content that doesn't invoke Cordova
-APIs.  The Windows Phone Developer Blog provides
-[helpful guidance](http://blogs.windows.com/windows_phone/b/wpdev/archive/2012/11/15/adapting-your-webkit-optimized-site-for-internet-explorer-10.aspx)
-on how to support IE along with comparable WebKit browsers.
+This guide shows how to set up your SDK development environment to build and deploy Cordova apps Windows 10 (Universal Windows Platform [= UWP], formerly known as Universal App Platform [= UAP]), Windows 8.1 and Windows Phone 8.1.  It shows how to use either shell tools to generate and build apps, or the cross-platform Cordova CLI. (See the [Overview](../../overview/index.html#development-paths) for a comparison of these development options.) This section also shows how to modify Cordova apps within Visual Studio. Regardless of [which approach](../../overview/index.html#development-paths) you take, you need to install the Visual Studio SDK, as described below.
 
-## Requirements and Support
+Cordova Windows on Windows 8.1 and Windows Phone 8.1 rely on Internet Explorer 11 as their rendering engine, so as a practical matter you can use IE's powerful debugger to test any web content that doesn't invoke Cordova APIs.  The Windows Phone Developer Blog provides [helpful guidance](https://blogs.windows.com/buildingapps/2012/11/15/adapting-your-webkit-optimized-site-for-internet-explorer-10/#qYPwLJDbYKToOveG.97) on how to support IE along with comparable WebKit browsers.
 
-To develop apps for Windows platform you need:
+## Requirements and Support
 
-- A Windows 8.1, 32 or 64-bit machine (_Home_, _Pro_, or _Enterprise_ editions)
-  with minimum 4 GB of RAM along with [Visual Studio 2015](http://www.visualstudio.com/downloads)
-  or Visual Studio 2013.  An evaluation version of Windows 8.1 Enterprise is
-  available from the
-  [Microsoft Developer Network](https://technet.microsoft.com/evalcenter/hh699156.aspx).
+To develop apps for Windows 10 you need:
 
-- For the Windows Phone emulators, Windows 8.1 (x64) Professional edition or higher,
-and a processor that supports <a href='https://msdn.microsoft.com/en-us/library/windows/apps/ff626524(v=vs.105).aspx#hyperv'>Client Hyper-V and Second Level Address Translation (SLAT)</a>.
+- Windows 10 (or Windows 8.1), 32- or 64-bit, along with [Visual Studio 2015](http://www.visualstudio.com/downloads) or higher.
```

Comment:
I think Cordova is not directly supported with Visual Studio 2019 since it dropped .jsproj support, as discussed in apache/cordova-windows#327.

Reply:
Good point, we can use this PR to include this information as well.

Reply:
Fixed and specified.